detekt/wrapper/src/test/java/io/buildfoundation/bazel/detekt/stream/WorkerStreamsTests.java

```diff
@@ -0,0 +1,80 @@
+package io.buildfoundation.bazel.detekt.stream;
+
+import bazel.worker.WorkerProtocol.WorkRequest;
+import bazel.worker.WorkerProtocol.WorkResponse;
+import io.reactivex.rxjava3.functions.Consumer;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class WorkerStreamsTests {
+
+    @Test
+    public void requestEmits() {
+        TestStreams streams = new TestStreams();
+        WorkerStreams workerStreams = new WorkerStreams.Impl(streams);
+
+        WorkRequest request24 = WorkRequest.getDefaultInstance()
+            .toBuilder()
+            .setRequestId(24)
+            .build();
+
+        WorkRequest request42 = WorkRequest.getDefaultInstance()
```

Comment:
Very minor nit: mind changing one of the two numbers? Having different digits (rather than the same in a different order) would be easier to distinguish